dissect/target/helpers/mount.py

```diff
@@ -3,7 +3,12 @@
 from functools import lru_cache
 from typing import BinaryIO, Optional
 
-from fuse import FuseOSError, Operations
+from dissect.util.feature import Feature, feature_enabled
+
+if feature_enabled(Feature.BETA):
+    from fusepy3.fuse import FuseOSError, Operations
```

Comment:
Semantics, but can't we make the import nearly the same? `from fuse3 import FuseOSError, Operations`?